ext4/internal/compactext4/compact.go

```diff
@@ -95,17 +95,17 @@ const (
 	inodeFirst        = 11
 	inodeLostAndFound = inodeFirst
 
-	blockSize               = 4096
-	blocksPerGroup          = blockSize * 8
+	BlockSize               = 4096
```

Comment:
Made `BlockSize` exported as this value needs to be the same both during ext4 creation and when generating the dm-verity information.